loops.js

```diff
@@ -32,8 +37,17 @@
 
 // Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over..
 // ==========================================
+oddOrEven = "";
```

Comment:
Begrijp niet helemaal waarom je deze regel geschreven hebt? Als declaratie mis ik dan nog `let`. Maar je hebt in principe geen aparte variabele nodig. Je kunt op regel 44 en 46 direct een `console.log` uitvoeren ipv pas op regel 49.